### ISSUE TITLE
Language Field in Field in figgy not pulling/viewing correct languages from MARC record

### ISSUE THREAD

**Issue by kelea99** [pulibrary/figgy#6170]
### Summary
Reported by Minjie Chen.  Figgy/DPUL is not reflecting the correct languages for an item record in the "Language" field in  figgy/DPUL. Slack with @tpendragon believes Language field being pulled from whatever the MARC code is for 008[35-37]:041a:041d. Confirmation from Minjie that pulling language data from 008/35-37, 041a, and 041d sounds straightforward, but that is not what Figgy is displaying, which is only English.  
Example record: [Figgy](https://figgy.princeton.edu/?f%5Bdisplay_subject_ssim%5D%5B%5D=Munsee+language%E2%80%94Readers) and [DPUL](https://dpul.princeton.edu/catalog?f%5Breadonly_subject_ssim%5D%5B%5D=Munsee+language%E2%80%94Readers&search_field=all_fields) identify them as in English only.

Test:
We chose [an item](https://figgy.princeton.edu/catalog/bb13ea71-acaf-4dc4-904d-1e01b8c1cda1) that originally was correctly viewing the languages. in this case Swedish and Delaware/ Munsee.  (showing Swedish/Delaware in Figgy before our test). 

> [from #digital_library] I thought I might as well try something uploaded in figgy that is :
> recent [post-alma migration and ingested 2022]
> Multi-lingual
> in Munsee/Delaware
> I found an item in figgy that reflects the same information as orangelight:https://figgy.princeton.edu/catalog/bb13ea71-acaf-4dc4-904d-1e01b8c1cda1
> Looking at the staff view of the languages:
> https://catalog.princeton.edu/catalog/9925251263506421/staff_view
> I see 2 041 fields:
> 041 1 a| del a| swe k| swe h| ger
> 041 7 a| swe a| umu 2| iso639-3
> 008 field: 008   860212s1696 sw bi 000 0 del^^

We refreshed the metadata in figgy to see what it would do, presuming it would only show one language. Something interesting happened. The languages did change, but to English only! Which is nowhere in the record.
 Confirmation from Minjie -   "eng" is definitely not appearing anywhere in the language codes of the Swedish-Munsee bilingual book. 
 
Questions:

- Is english a default placeholder value?
- Is solr changed/something changed that is viewing fine in orangelight but figgy is no longer pulling from/identifying the language codes and viewing in DPUL/Figgy?

### Impact 
_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
I want to test this in staging, but it would be good to know if this is affecting the entirety of the MARC ingests, if things are refreshed or new things ingested.  Ill ask @tpendragon today to work in staging.

### Priority recommendation

- [ ] asap
- [x] within the next 3 weeks
- [ ] PO will prioritize

## Sudden Priority Justification
_Required if "asap" or "within the next 3 weeks" is checked. Add "Sudden Priority" and "Maintenance/Research" labels_
If this is an issue affecting any/all newly ingested MARC in figgy/any all refreshed metadata in figgy, it affects the faceting for all items in DPUL.  It will make for a faulty search, perhaps wrongly showing items in a western language.

**Comment by escowles** [comment]
I know we made a number of changes in language display functionality in Orangelight/Bibdata, which is probably part of this. See https://github.com/pulibrary/bibdata/issues/2069 which is referenced by a few PRs.

**Comment by kelea99** [comment]
ok, so from what I see, Minjie's thought Re Figgy not being able to interpret the language code when it is taken from iso639 is possible/probable?

**Comment by escowles** [comment]
Yes, that is my guess — maybe if there are only ISO 639-3 languages, then there's a problem parsing them?

**Comment by minjiec** [comment]
Yes, I too suspect that ISO 639-3 language codes are trigging Figgy up.

Here is the context: [ISO 639-3 language codes](https://iso639-3.sil.org/code_tables/639/data) were applied only recently to a small number of our bibs [like these](https://catalog.princeton.edu/?f1=all_fields&f2=author&f3=title&f_inclusive%5Blanguage_facet%5D%5B%5D=Munsee&f_inclusive%5Blanguage_facet%5D%5B%5D=Wu+Chinese&op2=AND&op3=AND&q1=&q2=&q3=&range%5Bpub_date_start_sort%5D%5Bbegin%5D=&range%5Bpub_date_start_sort%5D%5Bend%5D=&search_field=advanced&sort=pub_date_start_sort+asc%2C+title_sort+asc%2C+score+desc). ISO 639-3 is much larger than "MARC Code List for Languages" so is able to match up with world languages, including indigenous languages and dialects, in greater granularity.
